### PR TITLE
basic transparency and code cleanup

### PR DESCRIPTION
**Pull request by deemfox**
(re?)implement transparency and remove useless code that caused canvas issues

### CODE REVIEW

**Review by robiot — APPROVED** [review]
Lgtm

File: main.py [hunk 1]
```diff
@@ -252,36 +255,13 @@ def get_board(self, access_token_in):
...
-                            }
-                        },
-                        "extensions": {},
-                        "operationName": "replace",
-                        "query": "subscription replace($input: SubscribeInput!) {\n  subscribe(input: $input) {\n    id\n    ... on BasicMessage {\n      data {\n        __typename\n        ... on FullFrameMessageData {\n          __typename\n          name\n          timestamp\n        }\n        ... on DiffFrameMessageData {\n          __typename\n          name\n          currentTimestamp\n          previousTimestamp\n        }\n      }\n      __typename\n    }\n    __typename\n  }\n}\n",
-                    },
-                }
-            )
-        )
-        ws.recv()
```
Comment (RomainL972) [inline main.py:277]: What did this code do ?

File: main.py [hunk 2]
```diff
@@ -252,36 +255,13 @@ def get_board(self, access_token_in):
...
 
         image_sizex = 2
         image_sizey = 1
 
         imgs = []
         already_added = []
-        for i in range(1, image_sizex * image_sizey + 1):
+        for i in range(0, image_sizex * image_sizey):
```
Comment (RomainL972) [inline main.py:264]: What does that do exactly ?

File: mappings.py [hunk 3]
```diff
@@ -1,4 +1,5 @@
 color_map = {
+    "#452A00": 0,  # transparent
```
Comment (RomainL972) [inline mappings.py:2]: Won't this make closest_color sometimes choose transparent as a color ?

**Review by RomainL972 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (deemfox) [reply main.py:277]: originally made requests for a link to the first canvas image, which was also being done by the for loop immediately below

File: mappings.py (hunk 3 above)
Comment (deemfox) [reply mappings.py:2]: yes. it should only be added to the name map

File: main.py (hunk 2 above)
Comment (deemfox) [reply main.py:264]: i missed that this loop was changed to fix the problem of getting the first canvas image twice, this undoes that change so the hard coded first request could still be removed.

Comment (RomainL972) [reply main.py:264]: Ok I think I get it. So now everything works ?

Comment (deemfox) [reply main.py:264]: yes. i have been running this version and everything seems to work

Comment (RomainL972) [reply main.py:264]: Ok, can you fix the conflicts so this can be merged ?

**Review by RomainL972 — APPROVED** [review]

**Pull request merged**